omnipath/_core/requests/_annotations.py

```diff
@@ -64,18 +66,24 @@ def get(
             Force the download of the entire annotations dataset. The full size of the data is ~1GB.
             We recommend to retrieve the annotations for a set of proteins or only from a few resources,
             depending on your interest.
+        wide
+            Pivot the annotations from a long to a wide dataframe format, reconstituting the format
+            of the original resource.
         kwargs
             Additional query parameters.
 
         Returns
         -------
         :class:`pandas.DataFrame`
-            A dataframe containing different gene/complex annotations.
+            A dataframe containing different molecule (protein, complex, gene, miRNA, small molecule) annotations.
+            If :arg:`wide` is `True` and the result contains more than one resource, a `dict` of dataframes
```

Comment:
:arg:\`wide\` cannot be interpreted in the docs, would just use \`\`if wise = True\`\`